example/android/gradle.properties

```diff
@@ -25,7 +25,7 @@ android.useAndroidX=true
 android.enableJetifier=true
 
 # Version of flipper SDK to use with React Native
-FLIPPER_VERSION=0.174.0
+FLIPPER_VERSION=0.182.0
```

Comment:
Is this needed? As we are not using flipper and it's getting deprecated

Reply:
I just followed RN upgrade helper 😄 I guess it will be soon removed in 0.73